build/tasks/build.js

```diff
@@ -11,7 +11,7 @@ module.exports = function( grunt ) {
 	var fs = require( "fs" ),
 		requirejs = require( "requirejs" ),
 		srcFolder = __dirname + "/../../src/",
-		rdefineEnd = /\}\);[^}\w]*$/,
+		rdefineEnd = /\}\s*?\);[^}\w]*$/,
```

Comment:
A very minor nit: the non-greediness doesn't matter here as `\)` doesn't match `\s` anyway, other regexps in this file don't use it.